submissions/kasionio/oop-exercise/index.js

```diff
@@ -0,0 +1,100 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kasionio/a-tiny-JS-world/tree/gh-pages
+   Web app: https://kasionio.github.io/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends = 'forever alone') {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying;
+      this.friends = friends;
```

Comment:
Can we re-use `addFriends` method here for consistency?
Besides within this class `friends` are treated as string in `constructor` and objects in `addFriends`. Confusing.